packages/plugingen/userop.json

```diff
@@ -0,0 +1,37 @@
+{
+  "chain": "arbitrum-sepolia",
+
+  "connection_config": {
+    "bundler": {
+      "rpcUrl": "https://arb-sepolia.g.alchemypreview.com/v2/fGnvtpbRdda2Tk4ewN5nWxoX0dscaV_4"
+    },
+    "public": {
+      "rpcUrl": "https://arb-sepolia.g.alchemy.com/v2/MlNZ2V-DTCdvd3jqSAfaf3yDMIBciA8F"
```

Comment:
definitely shouldn't have leaked this....

Reply:
ah removed. thought I added to gitignore.